### PR TITLE
Feature/16 on change function

### PR DESCRIPTION
**Pull request by jasrial**

### CODE REVIEW

File: src/components/TagRules.js [hunk 1]
```diff
@@ -7,66 +7,81 @@ import "./TagRules.css";
...
+
   const deleteRuleSet = index => {
+    console.log("deleteRuleSet", index);
     dispatch({
       type: "deleteRuleSet",
       index: index
     });
   };
 
+  console.log("TagRules ruleSets", ruleSets);
```
Comment (vlabs) [inline src/components/TagRules.js]: remove logs

File: src/reducers/ruleSetReducer.js [hunk 2]
```diff
@@ -20,6 +20,11 @@ const ruleSetReducer = (state, action) => {
         rules: removeArrayElement(state.rules, action.index)
```
Comment (vlabs) [inline src/reducers/ruleSetReducer.js]: this file can now be removed

File: src/components/TagRules.js (hunk 1 above)
Comment (jasrial) [reply src/components/TagRules.js]: b43670a8116d37d9d090f2f1e9aaff05bbbba514

File: src/reducers/ruleSetReducer.js (hunk 2 above)
Comment (jasrial) [reply src/reducers/ruleSetReducer.js]: 3eb07cb0f96b62dc4169f5465c19fe055316df2b

**Pull request merged**